### PR TITLE
Adding video,videolisting,routers,

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
-added videos details in backend
-added pages
-added basic routing to pages
-showing videolisting

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            </svg>
+          </button>
+        </form>
+      </div>
+      <div class="flex-row-center profile-logo-div">
+        <svg width="4rem" height="4rem" viewBox="0 0 32 32">
+          <path fill="#fbff00" d="M16 8a5 5 0 1 0 5 5a5 5 0 0 0-5-5Z"></path>
+          <path
+            fill="currentColor"
+            d="M16 2a14 14 0 1 0 14 14A14.016 14.016 0 0 0 16 2Zm7.992 22.926A5.002 5.002 0 0 0 19 20h-6a5.002 5.002 0 0 0-4.992 4.926a12 12 0 1 1 15.985 0Z"
```
Comment (manismk) [inline src/components/Navbar.jsx:48]: change class to className

File: src/components/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          </Link>
+        <div class="video-card-desc">
+          <h3 class="video-card-title">{title}</h3>
+          <Link class="channel-name white-color font-bold" to={channelLink} target="_blank">{channelName}</Link>
+          <div class="video-views">{views} views &nbsp; {uploaded}</div>
+        </div>
+        <svg width="3rem" height="3rem" viewBox="0 0 24 24">
+          <path
+            fill="#d6d6d6"
+            d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2s-2 .9-2 2s.9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2s2-.9 2-2s-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2s2-.9 2-2s-.9-2-2-2z"
```
Comment (manismk) [inline src/components/VideoCard.jsx:30]: change class to classname

**Review by manismk — APPROVED** [review]
Good naming convention and good folder structure

**Review by saketkum11 — APPROVED** [review]
- Clean code
- easy to understand.

**Pull request merged**